modules/ezcontent_node/modules/ezcontent_landing_page/config/install/core.entity_view_display.node.landing_page.default.yml

```diff
@@ -95,7 +95,9 @@ third_party_settings:
         System: {  }
         User: {  }
         core: {  }
-      blacklisted_blocks: {  }
+      blacklisted_blocks:
+        'Inline blocks':
+          - 'inline_block:paragraph_content'
```

Comment:
Since we are planning to remove `paragraph_content` block component, blacklisting/hiding it won't be a good solution as this is a configuration change and would need an unnecessary configuration update.